source/basic.tex

```diff
@@ -3988,7 +3988,7 @@
 currently installed new-handler function\iref{new.handler}, if any.
 \begin{note}
 \indextext{\idxcode{new_handler}}%
-A program-supplied allocation function can obtain the address of the
+A program-supplied allocation function can obtain the
```

Comment:
This part is non-normative and seems like an improvement. `std::new_handler` is defined as a pointer to a function, so what `std::get_new_handler` returns is a new_handler, not the address of one.

Reply:
Indeed, thanks. @lprv Feel free to send this one change as a separate PR if you like!